pages/api/auth/[id]/verify.tsx

```diff
@@ -0,0 +1,48 @@
+import { StatusCodes } from 'http-status-codes';
+import type { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
+import { confirmVerifyCode } from '../../../../lib/gateways/applications-api';
+import cookie from 'cookie';
+
```

Comment:
This file name is also a bit non-obvious. Maybe go long and call it `confirmVerifyCode` or `confirm-verify-code`.